assets/script/object.js

```diff
@@ -0,0 +1,235 @@
+            <p class="sub-item">
+              <img
+                src="${project.imageDesktop}"
+                alt=""
+                class="article-img"
+              />
+            </p>
+            <div class="article-content">
+              <h2 class="article-title">${project.title}</h2>
+              <h3>${project.company} <span>&bull; ${project.specialization} &bull; ${project.year}</span></h3>
+              <p class="article-text">
+              ${project.description1}
+              </p>
+              <ul class="article-tags">
+                <li class="article-tag ok">${project.technologies.tech1}</li>
+                <li class="article-tag">${project.technologies.tech2}</li>
+                <li class="article-tag">${project.technologies.tech3}</li>
+              </ul>
+              <button class="article-btn button_see_2" id="${project.id}">${project.details}</button>
+              
+            </div>
+          </article>
+        </section></a
+      >
+  `;
+  Worksection.append(cardDiv);
+  const btnSeeProject = document.getElementById(project.id);
+  btnSeeProject.addEventListener("click", () => {
+    console.log("Clic btn " + project.id);
+    //call our madal
+    showPopup(project);
+  });
+});
+
+function showPopup(pop) {
+  console.log(pop.title);
+  const body = document.querySelector(".main_body");
+  const modal = document.createElement("div");
+  modal.classList.add("modal");
+  modal.setAttribute("id", pop.id);
+  const popup = document.createElement("div");
+  popup.innerHTML = `
+  <div class="model model_2 model_3 model_4" id>
+<div class="model-main">
+  <div class="sub-div">
+    <div class="title-links-closeButton">
+      <div class="title-links-closeButton-title">
+        <h2>${pop.title}</h2>
+        <div class="icon-container">
+          <i class="fa-regular fa-x fa-x-button" id="desktop-id"></i>
+        </div>
+      </div>
+      <div class="title-links-closeButton-title-links">
+      <h2 class="article-title"></h2>
+        <li class="li-1"><a href="" class="detail-1">${pop.company}&bull;</a></li>
+        <li class="li-1"><a href="" class="detail-2">${pop.specialization} &bull;</a></li>
+        <li class="li-1"><a href="" class="detail-3">${pop.year}</a></li>
+      </div>
+    </div>
+    <div class="img-sub-div">
+      <img src="${pop.imageDesktop}" alt="sub-div1" class="img-height">
+    </div>
+    <div class="information-row-three">
+      <div class="p-information-row">
+        <p>${pop.description1}</p>
+      </div>
+      <div class="linkAndButtons">
+        <ul class="technologies-desktop-component tec-link">
+          <li><a href="" class="tecnologia_1 tec">${pop.technologies.tech1}</a></li>
+          <li><a href="" class="tecnologia_2 tec">${pop.technologies.tech2}</a></li>
+          <li><a href="" class="tecnologia_3 tec">${pop.technologies.tech3}</a></li>
+          <li><a href="" class="tecnologia_3 tec">${pop.technologies.tech4}</a></li>
+        </ul>

```

Comment:
👏👏 Great job on implementing the project description functionality! 🌟 However, there are a couple of improvements that I'd like to suggest:

- [Optional]🔄 For the technologies list, consider making it more dynamic by using a loop instead of limiting it to four technologies. In a real scenario, a project can have any number of technologies, so using a loop will allow you to display all the technologies without any limitations.

- [optional]🔗 Additionally, the technologies displayed currently do not have any redirection to other places. Since they only represent the technologies used in that project, using anchor tags for them might not be meaningful. Instead, you could consider adding project tags to describe the project in more detail.

By making these changes, your project description will become more flexible and user-friendly, allowing you to display all the necessary information without any constraints. Keep up the good work, and happy coding! 🚀💻